Cargo.toml

```diff
@@ -6,7 +6,7 @@ homepage = "https://kilt.io/"
 license-file = "LICENSE"
 readme = "README.md"
 repository = "https://github.com/KILTprotocol/kilt-node"
-version = "1.12.0-dev"
+version = "1.13.0-dev"
```

Comment:
Also the spec version need to be bumped to 11300 so that type augmentation works properly.

Reply:
Yes. you are right: [1af49a7](https://github.com/KILTprotocol/kilt-node/pull/579/commits/1af49a7b5569bab0799fb24fee9a4757df2533c4)

Reply:
Can you do it also for the DIP runtime templates?

Reply:
[0ebdac0](https://github.com/KILTprotocol/kilt-node/pull/579/commits/0ebdac0b8f5335b20f89cf201405ef1ad890c2ef)

Reply:
What's with all these changes? 😅

Reply:
rebased: [e0c4a3d](https://github.com/KILTprotocol/kilt-node/pull/579/commits/e0c4a3d6747b54641ab25c43afbb6b248ebfd2f5)